pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/utils/OperatorUtils.java

```diff
@@ -40,6 +45,8 @@ public class OperatorUtils {
     OPERATOR_TOKEN_MAPPING.put("*", "times");
     OPERATOR_TOKEN_MAPPING.put("/", "divide");
     OPERATOR_TOKEN_MAPPING.put("||", "concat");
+    SUPPORTED_EXCHANGE_TYPE.add(RelDistribution.Type.RANDOM_DISTRIBUTED);
+    SUPPORTED_EXCHANGE_TYPE.add(RelDistribution.Type.HASH_DISTRIBUTED);
```

Comment:
because today the only use for BROADCAST EXCHANGE is when join key is empty in PinotJoinExchangeNodeInsertRule.

When this happens, we make left join table local and broadcast right table to every server.

1) right table broadcast is a waste under this situation because we only need it for 1 server.
2) We have a bug for singleton broadcast. We should solve the local exchange optimization in a different way.